Fixes image saving behaviour

wschoenell opened the pull request:
After astroufsc/chimera#40 some controllers started to not work because of the lack of the  `.fits` file. This is due two reasons:

* The default `ImageRequest` compress parameters are defined to be `True` and `bz2`,  compressing images always by default.

* On the camera instrument, it always compress the files using bzip2, no matter if `compress` is set to `False` or `compress_format` is not `bz2` on the `ImageRequest`.

This PR fixes this behavior by adding sane defaults and fixing camera instrument.

wschoenell commented:
Tested here.

	(chimera)Williams-MacBook-Pro:~ william$ for compression in bz2 no gzip zip; do chimera-cam --compress=$compression; done
	======================================== 
	Taking 1 OBJECT frame[s] 
	Shutter: OPEN 
	Interval between frames: 0.000s 
	Cooling disabled. 
	Current CCD temperature: 20.501 oC 
	No binning 
	Full Frame 
	======================================== 
	synchronizing dome slit ... OK 
	======================================== 
	[001/001] [Thu Jul 23 13:37:53 2015] 
	exposing (1.000s) ... OK (took 1.112 s) 
	reading out and saving ... OK (took 3.897 s) 
	 (/Users/william/images/20150723/20150723-113757-0001.fits.bz2)  OK (took 2.783 s) 
	[001/001] took 3.899s 
	======================================== 
	Total time: 8.123s 
	======================================== 
	Thu Jul 23 13:37:57 2015 
	======================================== 
	======================================== 
	Taking 1 OBJECT frame[s] 
	Shutter: OPEN 
	Interval between frames: 0.000s 
	Cooling disabled. 
	Current CCD temperature: 20.451 oC 
	No binning 
	Full Frame 
	======================================== 
	synchronizing dome slit ... OK 
	======================================== 
	[001/001] [Thu Jul 23 13:38:04 2015] 
	exposing (1.000s) ... OK (took 1.109 s) 
	reading out and saving ... OK (took 3.819 s) 
	 (/Users/william/images/20150723/20150723-113808-0001.fits)  OK (took 2.707 s) 
	[001/001] took 3.820s 
	======================================== 
	Total time: 6.182s 
	======================================== 
	Thu Jul 23 13:38:08 2015 
	======================================== 
	======================================== 
	Taking 1 OBJECT frame[s] 
	Shutter: OPEN 
	Interval between frames: 0.000s 
	Cooling disabled. 
	Current CCD temperature: 20.476 oC 
	No binning 
	Full Frame 
	======================================== 
	synchronizing dome slit ... OK 
	======================================== 
	[001/001] [Thu Jul 23 13:38:13 2015] 
	exposing (1.000s) ... OK (took 1.114 s) 
	reading out and saving ... OK (took 5.073 s) 
	 (/Users/william/images/20150723/20150723-113818-0001.fits.gz)  OK (took 3.957 s) 
	[001/001] took 5.075s 
	======================================== 
	Total time: 7.246s 
	======================================== 
	Thu Jul 23 13:38:18 2015 
	======================================== 
	======================================== 
	Taking 1 OBJECT frame[s] 
	Shutter: OPEN 
	Interval between frames: 0.000s 
	Cooling disabled. 
	Current CCD temperature: 20.653 oC 
	No binning 
	Full Frame 
	======================================== 
	synchronizing dome slit ... OK 
	======================================== 
	[001/001] [Thu Jul 23 13:38:23 2015] 
	exposing (1.000s) ... OK (took 1.109 s) 
	reading out and saving ... OK (took 5.052 s) 
	 (/Users/william/images/20150723/20150723-113828-0001.fits.zip)  OK (took 3.969 s) 
	[001/001] took 5.082s 
	======================================== 
	Total time: 7.084s 
	======================================== 
	Thu Jul 23 13:38:28 2015 
	======================================== 

![screen shot 2015-07-23 at 13 39 59](https://cloud.githubusercontent.com/assets/1873811/8849516/9b2ce2e8-3140-11e5-80c9-25dd91da28a0.png)

phsilva commented on src/chimera/controllers/imageserver/imagerequest.py:
```diff
@@ -27,8 +27,8 @@ def __init__(self, **kwargs):
                     'filename': '$DATE-$TIME',
-                    'compress': True,
-                    'compress_format': "BZ2",
+                    'compress': False,
```
If we can pass compress: "NO", can't see the point of having a boolean flag for it as well, looks redundant. Why not simply 'compress' begin an Enum (NO, BZ2, GZIP, ZIP)?

wschoenell commented:
I have added the changes suggested by @phsilva and moved the name file resolution method from the `Image` object to `chimera-cam`. The way that it was being done was relying on the existence of the file. This was sometimes making `chimera-cam` print the wrong filename.

wschoenell commented:
Last commits tested as above and everything ok!
